runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -105,6 +105,8 @@ public void resetCaches() {
         readCache.invalidateAll();
     }
 
+    ReadOptions defaultReadOptions = new ReadOptions(false, true, true, true);
```

Comment:
Can you please use the builder here.

Reply:
Done.

Reply:
Was addressed in https://github.com/CorfuDB/CorfuDB/pull/2055/files will resync the branches

Reply:
Done.